src/main/java/xyz/groundx/caver_ext_kas/rest_client/io/swagger/client/api/kip37/model/Kip37FeePayerOption.java

```diff
@@ -0,0 +1,115 @@
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+package xyz.groundx.caver_ext_kas.rest_client.io.swagger.client.api.kip37.model;
+
+import java.util.Objects;
+import java.util.Arrays;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.io.IOException;
+import xyz.groundx.caver_ext_kas.rest_client.io.swagger.client.api.kip37.model.Kip37FeePayerOptionUserFeePayer;
+/**
+ * Kip37FeePayerOption
+ */
+
+
+public class Kip37FeePayerOption {
+  @SerializedName("enableGlobalFeePayer")
+  private Boolean enableGlobalFeePayer = null;
+
+  @SerializedName("userFeePayer")
+  private Kip37FeePayerOptionUserFeePayer userFeePayer = null;
+
+  public Kip37FeePayerOption enableGlobalFeePayer(Boolean enableGlobalFeePayer) {
+    this.enableGlobalFeePayer = enableGlobalFeePayer;
+    return this;
+  }
+
+   /**
+   * A boolean value of whether KAS Global FeePayer is used. The default value is &#x60;true&#x60;.
+   * @return enableGlobalFeePayer
+  **/
+  @Schema(description = "A boolean value of whether KAS Global FeePayer is used. The default value is `true`.")
+  public Boolean isEnableGlobalFeePayer() {
```

Comment:
Is there any specific reason you implemented `isEnableGlobalFeePayer` and not `getXXX`?

Reply:
This code just reproduced by swagger codegen using KIP-37 API open API spec.